src/components/10-atoms/spinner/README.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+Adds a loading spinner that can be used while something is loading.
+
+```bash
+npm install @axa-ch/spinner
+```
+
+```js
+import '@axa-ch/spinner';
+...
+<axa-spinner></axa-spinner>
+```
+
+## Properties
+
+### Variant
+
+| Attribute                   | Details              |
+| --------------------------- | -------------------- |
+| `size=false`                | Size of the spinner  |
```

Comment:
This is a boolean attribute, so we should just write `size` and explain in the details, what it's doing. Anyways, since you made a boolean out of it, you should rename it to `small` in the code base. So that the variable tells you what it's going to do. `Size` does not tell you that it is a boolean. When I read that I expect that I could set some numbers.